hassio-google-drive-backup/backup/static/picker.jinja2

```diff
@@ -78,81 +78,8 @@
     </div>
     </div>
   </main>
-  <footer class="ha-blue page-footer">
-    <div class="container">
-      <div class="row">
-        <div class="col l3 s12">
-          <h5 class="accent-text">Connect</h5>
-          <ul>
-            <li><a class="accent-text" href="https://github.com/sabeechen/hassio-google-drive-backup">Project Github</a>
-            </li>
-            <li><a class="accent-text" href="https://github.com/sabeechen/hassio-google-drive-backup/issues">Issue
-                Tracker</a></li>
-            <li><a class="accent-text" href="https://materializecss.com">Materialize</a></li>
-          </ul>
-        </div>
-        <div class="col l6 s12">
-          <h5 class="accent-title">Who did this?</h5>
-          <p class="accent-text">I'm an individual who works on projects like this in my spare time. If you
-            like it and would like to support me, you can star my projects on <a
-              href="https://github.com/sabeechen">Github</a> or just throw money at me              
-            <style>
-              .bmc-button img {
-                width: 25px !important;
-                margin-bottom: 1px !important;
-                box-shadow: none !important;
```

Comment:
I am unsure if this `<style>` tag is important or not, but I've removed it and used the same footer as all the other pages.

Reply:
It is, but with these changes I'd like to handle it a different way.  The picker webpage is only exposed on the externally hosted part of this addon (eg [https://habackup.io/drive/picker?bg=FFFFFFFF&ac=0000FFFF](https://habackup.io/drive/picker?bg=FFFFFFFF&ac=0000FFFF)), and it actually is set up to correctly render the background/accent colors the user users locally.  The style sheets and javascript on that page get broken with these changes, but I'm happy to fix that myself because it would be difficult to actually test without uploading to my backend.  I'm happy to leave that part broken for now (since this change is worthwhile as is) and I'll fix it up before I make another release.